hbase-server/src/main/resources/hbase-webapps/master/table.jsp

```diff
@@ -77,6 +77,7 @@
 <%@ page import="org.apache.hadoop.hbase.shaded.protobuf.generated.QuotaProtos.Quotas" %>
 <%@ page import="org.apache.hadoop.hbase.shaded.protobuf.generated.QuotaProtos.SpaceQuota" %>
 <%@ page import="java.util.stream.Collectors" %>
+<%@ page import="org.apache.hadoop.hbase.rsgroup.RSGroupAdminClient" %>
```

Comment:
On master branch please just use Admin instead of RSGroupAdminClient?

Reply:
> On master branch please just use Admin instead of RSGroupAdminClient?

I analyzed that only RSGroupAdminClient supports the moveTables method, and Admin does not support the moveTables method

Reply:
IIRC, on master, the RSGroup information is stored as a table property, you just need to modify the table's property, then the table will be moved to the new rs group.